addons/mail/static/src/discuss/core/web/discuss_core_web_service.js

```diff
@@ -102,6 +102,8 @@ export class DiscussCoreWeb {
                                 folded: thread.state === "folded",
                             });
                         }
+                    } else if (chatWindow?.hidden && data.serverFoldState === "open") {
+                        this.chatWindowService.makeVisible(chatWindow);
```

Comment:
I don't think we should change the hidden state from the bus either actually. The user already selected which to display, it should be a user action to change it.